patches/u-boot/series

```diff
@@ -13,3 +13,4 @@ arch-powerpc-85xx-always-call-board_reset.patch
 driver-support-new-broadcom-phys.patch
 platform-onie-common-env.patch
 platform-common-env.patch
+arch-powerpc-fix-mpc8541-ss_en_bit.patch
```

Comment:
For the global series file I try to keep it organized by putting all the arch-XXX changes together in one place, with the ONIE specific changes at the end.  For example for your change the series file should look like:
```
git-ignore.patch
feature-config-repeatable.patch
feature-dhcp-options.patch
feature-save-default-env-on-bad-crc.patch
feature-populate-serial-number.patch
feature-sys-eeprom-tlv.patch
feature-fdt-environment-size.patch
arch-powerpc-define-85xx-sdhc-polarity-select.patch
arch-powerpc-fix-p2020-i2c-clock.patch
arch-powerpc-fix-fsl-ddram-cfg.patch
arch-powerpc-85xx-always-call-board_reset.patch
arch-powerpc-fix-mpc8541-ss_en_bit.patch    <---- new patch here
driver-support-new-broadcom-phys.patch
platform-onie-common-env.patch
platform-common-env.patch
```